scala-thoughts.md

```diff
@@ -0,0 +1,5 @@
+I really like scala so far. So far it seems like it gives me everything I liked about functional Python and Haskell with the benefits of a static type system over Python and flexibility over Haskell. Parts of it remind me of Java and I feel like they're mostly vestiges from it being made in Java, and they don't really affect the feel fo the language very much. I actually don't really have any qualms at all right now; it seems like a good language. 
+
```

Comment:
Your comment about Scala reminding you of Java is really interesting to me. I didn't realize it until now, but I feel the same way. I'm not sure if it's something about how the different design patterns are implemented or just things about the syntax/function structure, but Scala definitely felt more familiar to me (AKA Java-like) than other languages I've learned.